api-tests/local_newman.sh

```diff
@@ -5,13 +5,42 @@
 #     export GWELLS_API_BASE_URL="http://localhost:8000/gwells/registries"
 #     export GWELLS_API_TEST_USER="testuser"
 #     export GWELLS_API_TEST_PASSWORD="secret"
+#     export GWELLS_API_TEST_AUTH_SERVER=""
+#     export GWELLS_API_TEST_CLIENT_ID=""
+#     export GWELLS_API_TEST_CLIENT_SECRET"""
 # - Create Django test user (requires env variables from previous step):
 #     python manage.py createtestuser
 # - Install newman:
 #     npm install -g newman
 # - Run script:
 #     ./local_newman.sh
 
+
+if [ -z ${GWELLS_API_TEST_USER+x} ]; then 
```

Comment:
If this is going to be updated regularly it might be better to iterate though a list.  Otherwise this is looking thorough!

E.g. 
```
THINGS=(
	"GWELLS_API_TEST_USER"
	"GWELLS_API_TEST_PASSWORD"
	"GWELLS_API_BASE_URL"
	"GWELLS_API_TEST_AUTH_SERVER"
	"GWELLS_API_TEST_CLIENT_SECRET"
)
for t in ${THINGS[@]}
do
	[  ! -z $t ] || echo "Unset: $t" && exit
done

Reply:
It's less code, so I like it - I'm not quite the bash expert though, so I've done this:

```ENV_VARS=(
    "GWELLS_API_TEST_USER"
    "GWELLS_API_TEST_PASSWORD"
    "GWELLS_API_BASE_URL"
    "GWELLS_API_TEST_AUTH_SERVER"
    "GWELLS_API_TEST_CLIENT_ID"
    "GWELLS_API_TEST_CLIENT_SECRET"
)

for env_var in ${ENV_VARS[@]}
do
    if [ -z ${!env_var+x} ]; then 
        echo "$env_var is unset"
        exit
    fi
done
```